renderer/brave_content_renderer_client.cc

```diff
@@ -147,8 +148,9 @@ void BraveContentRendererClient::RunScriptsAtDocumentStart(
   auto* observer =
       cosmetic_filters::CosmeticFiltersJsRenderFrameObserver::Get(render_frame);
   // Run this before any extensions
-  if (observer)
+  if (observer) {
```

Comment:
Wow, that's the wrong observer: `CosmeticFiltersJsRenderFrameObserver`. Please fix it.

Reply:
I wonder why all the builds are green..
@antonok-edm Do we have a test for cosmetic filters in an incognito window?